managedtenants/utils/ocm.py

```diff
@@ -111,7 +111,11 @@ def list_sku_rules(self):
 
     def add_addon(self, metadata):
         addon = self._addon_from_metadata(metadata)
-        return self._post("/api/clusters_mgmt/v1/addons", json=addon)
+        # Update Tooling to point to new API MTSRE-601
+        self._post(
+            "/api/addons_mgmt/v1/addons", json=addon)
+        return self._post(
+            "/api/clusters_mgmt/v1/addons", json=addon)
```

Comment:
I see that for every change you have followed the order:
1. addons_mgmt endpoint
2. clusters_mgmt endpoint

Now, as we addressed above, this can compromise atomicity say, when the Step 1 succeeds and Step 2 fails i.e. the addons_mgmt endpoint will be updated with new data whereas cluster_mgmt one will be left with stale.

In theory, we can swap that so that in the worst case, cluster_mgmt ends up with new data and addons_mgmt ends up with stale.

I believe we should have a chat with service-dev folks to see which of the above two situations would be more reliable.

If the service-dev folks have implemented some sort of reconciliation between addons_mgmt and cluster_mgmt (say, addons_mgmt periodically syncs itself to clusters_mgmt), then we can go with
Step 1. clusters_mgmt
Step 2. addons_mgmt

Reply:
Thank you Yash.
How do I contact service-dev folks to ask above questions?

Reply:
we can reach out to #service-development